salt/passenger/init.sls

```diff
@@ -0,0 +1,32 @@
+{% from 'lib.sls' import apache %}
+
+include:
+  - apache
+
+# https://www.phusionpassenger.com/library/install/apache/install/oss/bionic/
+passenger dependencies:
+  pkg.installed:
+    - pkgs:
+      - dirmngr
+      - gnupg
+      - python3-gnupg
```

Comment:
I didn't see `python3-gnupg` in https://www.phusionpassenger.com/library/install/apache/install/oss/bionic/ What is it needed for?

Reply:
In a previous local iteration of this I was importing the RVM GPG key directly and the salt GPG key integration requires `python3-gnupg`.
Now the code uses the apt repository I believe we can remove this. I will test, confirm and commit.